pom.xml

```diff
@@ -113,6 +114,69 @@
 				<version>${javax.servletVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.springframework.ws</groupId>
+				<artifactId>spring-ws-core</artifactId>
+				<version>${springWsVersion}</version>
+				<scope>compile</scope>
+			</dependency>
+
+			<dependency>
```

Comment:
Where do you use all of these libraries below? Is every one of them necessary?

Reply:
All of them are required to use spring-ws-core with compile scope. Without them there will be org.springframework.beans.FatalBeanException thrown, during Refreshing Context. you can find the list here: https://mvnrepository.com/artifact/org.springframework.ws/spring-ws-core/2.4.2.RELEASE . As you can see in the link javax.servlet-api is not rwquired when using compile scope, however without it Refreshing Context also fails, so I decided to leave it here.